<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabWidget</class>
 <widget class="QTabWidget" name="TabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;asd&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tab_1">
   <attribute name="title">
    <string>Databse search</string>
   </attribute>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>961</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="doc_btn">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>260</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="query_filename">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="candidates_btn">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Search candidates</string>
     </property>
    </widget>
    <widget class="QListWidget" name="candidates_list">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>120</y>
       <width>251</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="score_list">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>120</y>
       <width>251</width>
       <height>151</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>qwe</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>New Item</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>New Item</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>qwe</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="load_btn1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Load file</string>
     </property>
    </widget>
    <widget class="QPushButton" name="score_btn">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>40</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Score</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>961</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_2">
   <attribute name="title">
    <string>Local search</string>
   </attribute>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>961</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>961</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="doc_btn_2">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>260</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Load file 2</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_7">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>150</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_8">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>120</y>
       <width>251</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="load_btn2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Load file 1</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>40</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Score</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
